recipe/bld.bat

```diff
@@ -16,7 +16,11 @@ if "%ARCH%"=="64" (
 )
 
 cd PCbuild
-call build.bat --pgo -m -e -v -p %PLATFORM%
+REM try to use 1.5 times of the logical cores
```

Comment:
This doesn't make much sense, generally, TBH.
I'm just trying to see if this works and if it affects the build time slightly.
The builds timeout "just before the finish line", hence this desperate attempt.
If this has not impact, I'll remove it again and we can try to merge this and just re-run the build jobs until we get lucky, I guess?!

Reply:
The build times (before PGO) are 4-6 minutes, i.e., a large error of margin. I'm not sure if any potential effect from this could be recognized at all..

Reply:
We dropped PGO from the 3.8 and 3.7 branches due to the configuration and resource limits on the CI machines. We could do that here as well.

Reply:
Oh, I thought the PGO issues "sorted it self out" at some time. Didn't know they were still disabled.
Okay, I'll let the current CI job (at least partially) finish (out of curiosity) and then remove PGO.